onadata/apps/api/tools.py

```diff
@@ -121,7 +121,10 @@ def create_organization_object(org_name, creator, attrs=None):
         first_name=first_name,
         last_name=last_name,
         email=email,
-        is_active=True)
+        is_active=getattr(
+            settings,
+            'ORGANIZATION_ACTIVE_STATUS_ON_CREATE',
```

Comment:
I would recommend `ORG_ON_CREATE_IS_ACTIVE`.

Reply:
Alright let me change the naming.

Reply:
Done